apps/functions/applications-command-handlers/deadline-submission-command/index.js

```diff
@@ -8,7 +8,7 @@ const { submissionsContainer } = config;
 /**
  *
  * @param {import('@azure/functions').Context} context
- * @param {import('@pins/api/src/message-schemas/commands/new-deadline-submission').DeadlineSubmission} msg
+ * @param {import('pins-data-model').Schemas.DeadlineSubmission} msg
```

Comment:
Just want to confirm there is no difference between the message definitions for these commands?
If there are, this may be a breaking change if Applications FO sends data based on old definition files.